core/trino-main/src/main/java/io/trino/sql/analyzer/StatementAnalyzer.java

```diff
@@ -1866,7 +1866,13 @@ protected Scope visitAliasedRelation(AliasedRelation relation, Optional<Scope> s
             if (relation.getColumnNames() != null) {
```

Comment:
These seem unnecessary. We don't use hard-wrapping. If your IDE is changing them upon reformatting, you may want to update your IDE settings.

The general rule is: anything that is relevant to make sense of the code should fit within view in a typical screen (e..g, a laptop screen), but non-essential things can extend beyond -- for example, the elements making up an exception message, which don't affect the flow of code or logic.